SparkPost/2web2ui
# SD-748 - IP Auto Warmup

### What Changed
 - What changes did you make to satisfy the AC?
     - Allow ip warmup and stage to be selected/modified

### How To Test
- [ enable this option to your account](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md#give-access-to-your-feature): `ip_auto_warmup`. it's enabled on `appteam` on staging. remember to submit whole `ui` stanza with new/updated values. 
- enable/disable: No special instructions. Just check the box on/off. 
- stage: Use control endpoint to set higher stage. See https://msys.slack.com/archives/CFFCXGAE7/p1553099114072900
